### ISSUE TITLE
[tests] Use official images

### ISSUE THREAD

**Issue by nemesisdesign** [openwisp/ansible-openwisp2#179]
I suspect that the images we use for testing in the travis-ci build are not exactly identical to the official images our users download, so we cannot always replicate the same problems they have.

**Comment by atb00ker** [comment]
Two immediate reasons come to mind:
1. Our images might be outdated, @hispanico https://gitlab.com/ninuxorg/docker/ansible-debian (They are one year old)
2. Here are the changes in the image that are made before testing: https://gitlab.com/ninuxorg/docker/ansible-debian/-/blob/master/Dockerfile.debian9

**Comment by nemesisdesign** [comment]
@atb00ker what do you think we can do? Can we use some kind of latest official docker image for the distros we support? So we get the virgin system and we run the tests on that, we'll be able to spot all these errors that now we don't catch.

**Comment by atb00ker** [comment]
@nemesisdesign 

I think that before taking any action, it's worth investigating more into it, to specially find the reason that the mentioned error was not spotted in travis.

> Can we use some kind of latest official docker image for the distros we support

Certainly possible, I am not sure how to trigger the builds on Gitlab, but a trigger to rebuild images in the registry may be the key, *or* If I remember correctly, before gsoc 2019, we also discussed moving images to docker hub, we can do that as well and use docker hub auto build to build images periodically.

> we get the virgin system and we run the tests on that, we'll be able to spot all these errors that now we don't catch.

However, **if** the reason turns out to be the changes in the images, about two years ago, I created this: https://github.com/atb00ker/docker-ansible-openwisp2-development
Here, each image example debian 10: https://github.com/atb00ker/docker-ansible-openwisp2-development/blob/master/Dockerfile.debian10
only installs systemd and runs ansible-openwisp on the host (travis) instead of docker image, this **might** be helpful.
